css-view-transitions-2/Overview.bs

```diff
@@ -1097,6 +1114,8 @@ When setting up the transition pseudo-element for a [=captured element=] |captur
 		1. Let |parentGroup| be the ''::view-transition-group()'',
 			with its [=view transition name=] set to |capturedElement|'s [=containing group=].
 
+		1. Multiply |group|'s 'transform' by the inverse of |parentGroup|'s 'transform'.
```

Comment:
Same here.

Reply:
Done

Reply:
Sorry, I don't see the edits...

Reply:
Oops, pressed resolve instead of commit. Good now